src/api/db/migrate/20161122121211_move_comments_data.rb

```diff
@@ -0,0 +1,16 @@
+class MoveCommentsData < ActiveRecord::Migration[5.0]
+  def change
+    ActiveRecord::Base.transaction do
+      Comment.inheritance_column = nil
+      Comment.find_each(batch_size: 5000) do |comment|
+        /\AComment(?<type>.+)/ =~ comment.type
+        raise "Uncorrected type for comment with id #{comment.id}" unless ['Project', 'Package', 'BsRequest'].include? type
```

Comment:
This will stop the migration with some data migrated and other data not, not so ideal :-/

Why do you think this can happen?